user_management/doc/User_management.md

```diff
@@ -362,3 +362,40 @@ Might be extended to return more information in the future, if needed
 
 
 </details>
+
```

Comment:
line 325 is missing `</details>`.

Reply:
[7ff82cd](https://github.com/tdameros/42-transcendence/pull/58/commits/7ff82cd82f8ebfe6bb8ce481fd92b836ac1abd82)